### DISCUSSION TITLE
［委托］stick knock summer车队发布紧急委托

### DISCUSSION THREAD

**Discussion by Prance36** [share121/inter-knot#5550] [公告]
_**Stick Knock Summer**_车队在物资转运途中遭遇突发空洞灾害，目前车队人员均已全部撤离，但运输车及车上的两台赛车目前仍无下落。现发布紧急委托，寻求绳匠的帮助。

**Comment by Quatfolia** [comment]
这个委托我接不了（）

**Comment by RWHwaxili** [comment]
对不起，做不到

**Comment by fzdqz** [comment]
这个委托我接了

**Reply by KELERSAMA** [reply]
传奇钓鱼佬怎么当传奇绳斤来了

**Reply by fzdqz** [reply]
> 传奇钓鱼佬怎么当传奇绳斤来了

钓鱼是因为没有委托接了，绳匠才是本职工作
